idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -37,6 +41,16 @@ export default class PageManager {
       const Ctor = this.pages[page];
       const newPage = new Ctor(opts);
 
+      if (newPage.headerScroll) {
```

Comment:
Same basic comment I had from the other PR: This might work better as a single subview on the `base-view`, using the [built-in ampersand-view subview handling](https://github.com/AmpersandJS/ampersand-view#subviews-viewsubviews). That handles the DOM lookup, rendering, and appending. Then, let the view handle the logic of when to respond to scrolling based on options from the parent view or some other hint.